vite/vite.w

```diff
@@ -0,0 +1,139 @@
+bring cloud;
+bring sim;
+bring util;
+bring fs;
+bring "./find-port.w" as find_port;
+
+struct SimViteProps {
+  root: str;
+  env: Map<str>;
+  cli: str;
+  generateTypeDefinitions: bool;
+}
+
+class SimVite {
```

Comment:
```suggestion
class Vite_sim {
```

(this is the weird convention we had for this)

Reply:
Fair enough.